packages/wp-source/src/libraries/__tests__/populate.tests.ts

```diff
@@ -74,6 +74,7 @@ describe("populate", () => {
           link: "https://test.frontity.io/category/cat-1/",
           slug: "cat-1",
           taxonomy: "category",
+          count: 1,
```

Comment:
I've added the `count` here so that we can test if this property was actually updated and not just added to the state.